autogen.sh

```diff
@@ -1,21 +1,21 @@
 #!/bin/sh
 set -exu
 
-if ! type autoconf >/dev/null 2>/dev/null ; then
+if ! type autoconf >/dev/null 2>&1 ; then
```

Comment:
is this more portable? I thought `2>&1` was bash specific syntax

Reply:
It's actually the standard syntax for Bourne shell and the most portable across `ksh`, `zsh`, `bash`, etc.

Reply:
See for example [sh(1) man page](https://docs.oracle.com/cd/E86824_01/html/E54763/sh-1.html) or [_Bourne Shell Scripting_](https://en.wikibooks.org/wiki/Bourne_Shell_Scripting/Files_and_streams#Redirecting_standard_error_.28and_other_streams.29)Wikibook.